cli/src/lib/findFlowRoot.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import {searchUpDirPath} from "./fileUtils";
+import {fs, path} from './node';
+
+// Find the project root
+export default async function (start: string): Promise<?string> {
```

Comment:
So #289 just landed, which adds this to one of the commands: https://github.com/flowtype/flow-typed/blob/master/cli/src/commands/install.js#L247

If you rebase, would you mind updating that to just use this utility module?